src/OrbitQt/orbitsamplingreport.cpp

```diff
@@ -98,38 +105,21 @@ void OrbitSamplingReport::Initialize(orbit_data_views::DataView* callstack_data_
     uint32_t thread_id = report_data_view.GetThreadID();
     // Report any thread that contains more than 5% unwinding errors.
     constexpr double kUnwindErrorNoticeThreshold = 0.05;
-    double unwind_error_ratio = sampling_report_->ComputeUnwindErrorRatio(thread_id);
     if (sampling_report_->ComputeUnwindErrorRatio(thread_id) >= kUnwindErrorNoticeThreshold) {
-      auto* notice_box = new QWidget(tab);
-      notice_box->setStyleSheet(
-          ".QWidget { border-radius: 5px; border: 1px solid palette(text); background:"
-          " rgba(255,0,0, 10%); } "
-          "QPushButton:focus { border-color: rgba(255, 0, 0, 20%); }");
-      auto* notice_box_layout = new QGridLayout(notice_box);
-
+      auto* notice_box = new orbit_util_widgets::NoticeWidget(tab);
       std::string thread_tab_name = report_data_view.GetName();
       std::replace(thread_tab_name.begin(), thread_tab_name.end(), '\n', ' ');
-      auto* notice_label = new QLabel(QString::fromStdString(absl::StrFormat(
-                                          "%.2f%% of callstack samples in %s are unwinding errors.",
-                                          unwind_error_ratio * 100, thread_tab_name)),
-                                      notice_box);
-      notice_box_layout->addWidget(notice_label, 0, 0);
-      auto* notice_button = new QPushButton(notice_box);
-      notice_button->setText("Hide");
-      QObject::connect(notice_button, &QPushButton::clicked, notice_box,
+      double unwind_error_ratio = sampling_report_->ComputeUnwindErrorRatio(thread_id);
+      std::string label = absl::StrFormat("%.2f%% of callstack samples in %s are unwinding errors.",
+                                          unwind_error_ratio * 100, thread_tab_name);
+      notice_box->Initialize(label, "Hide", Color{255, 0, 0, 26});
```

Comment:
maybe here as well
```suggestion
      const Color kRed{255, 0, 0, 26};
      notice_box->Initialize(label, "Hide", kRed);
```

Or maybe even at the beginning of the file. At one point in the future I want to put all colors into one config file that is somewhere.. this would probably help a little

Reply:
added the colors to the top of the file